api/src/test/java/org/openmrs/util/OpenmrsUtilUnitTest.java

```diff
@@ -137,4 +147,128 @@ public void join_shouldReturnListElementsJoinedBySeparatorIfGivenSeparatorIsNull
 
 		assertThat(OpenmrsUtil.join(moduleList, null), is("module1nullmodule2"));
 	}
+
+	@Test
+	public void isStringInArray_shouldPassIfStringIsInArray() {
+
+		String[] array = new String[]{"element1","element2","element3"};
+
+		assertTrue(OpenmrsUtil.isStringInArray("element1", array));
+	}
+
+	@Test
+	public void isInNormalNumericRange_shouldNotAllowNullValuesAsLowerAndHigherNormal() {
+
+		ConceptNumeric concept = new ConceptNumeric();
+		concept.setHiNormal(null);
+
+		assertFalse(OpenmrsUtil.isInNormalNumericRange(5.67f, concept));
+	}
+
+	@Test
+	public void isInNormalNumericRange_shouldPassIfValueWithinRange() {
+
+		ConceptNumeric concept = new ConceptNumeric();
+		concept.setHiNormal(10.34d);
+		concept.setLowNormal(3.67d);
+
+		assertTrue(OpenmrsUtil.isInNormalNumericRange(5.67f, concept));
```

Comment:
can you please also add tests for when `hiNormal or lowNormal are not in range`, and edge cases hiNormal is equal, lowNormal is equal

when writing the tests please generate coverage with your ide and look at the hits on if statements/boolean expressions to make sure you test all cases. especially edge cases are often forgotten and cause bugs